dbc_member.rb

```diff
@@ -0,0 +1,32 @@
+class DbcMember
+  attr_reader :age, :salary, :phase, :performance_rating, :target_raise
+  attr_accessor :name
```

Comment:
Do you want all of these in all `DbcMembers`?  e.g. should student have a `performance_rating`?